sonar-ws/src/main/java/org/sonarqube/ws/client/issue/IssuesService.java

```diff
@@ -207,4 +214,17 @@ public SearchWsResponse search(SearchWsRequest request) {
       Issues.Operation.parser());
   }
 
+
+  public Issues.Operation setTags(String issue, String... tags) {
+    return call(new PostRequest(path(ACTION_SET_TAGS))
+        .setParam(PARAM_ISSUE, issue)
+        .setParam(PARAM_TAGS, Arrays.stream(tags).collect(Collectors.joining(","))),
+      Issues.Operation.parser());
+  };
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)